common/middleware/ensure-current-location.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function ensureCurrentLocation({
```

Comment:
this may benefit, readability wise, from having the `module.exports` at the bottom